app/src/components/network/PageBlock.vue

```diff
@@ -1,12 +1,11 @@
 <template>
   <TmPage
     data-title="Block"
-    :managed="true"
     :loading="this.$apollo.queries.block.loading"
     :error="error"
-    hide-header
+    class="readable-width"
   >
-    <template slot="managed-body">
+    <template v-if="!this.$apollo.queries.block.loading">
```

Comment:
managed-body does this already plus it also hides on error, right?

Reply:
What does `managed-body` actually do?

Reply:
`managed-body` is the content of the page that shows after everything is loaded and no errors appeared. It is defined as a slot in TmPage

Reply:
Oh thank you. Now I found the code for it: `<slot v-else name="managed-body" />`

Yes, and it is already doing the loading check as you said (line 22)

Reply:
true, thanks.